hack/build.sh

```diff
@@ -66,6 +66,8 @@ for dir in ${dirs}; do
     *)
       if [ "${OS}" == "centos7" ]; then
         NAMESPACE="centos/"
+      elif [ "${OS}" == "fedora" ]; then
+        NAMESPACE="fedora/"
```

Comment:
Ok. Later we could parse it from the `FROM` instruction inside the Dockerfile I suppose.